web/lib/roadmap/roadmapBuilder.ts

```diff
@@ -25,6 +26,29 @@ export const buildRoadmap = async ({
   return convertToRoadmap(roadmapBuilder);
 };
 
+const generateIndustryRoadmap = async (
+  builder: RoadmapBuilder,
+  industryId: string,
+  addOns: string[],
+  modifications: string[]
+): Promise<RoadmapBuilder> => {
+  let industryRoadmap: IndustryRoadmap;
+  if (process.env.NODE_ENV === "test") {
+    industryRoadmap = (await import(`@/lib/roadmap/fixtures/industries/${industryId}.json`))
+      .default as IndustryRoadmap;
+  } else {
+    industryRoadmap = (await import(`../../roadmaps/industries/${industryId}.json`))
+      .default as IndustryRoadmap;
+  }
+
+  addTasksFromAddOn(builder, industryRoadmap.addOns);
+  await applyAddOns(builder, ["generic-tasks", ...addOns]);
```

Comment:
Small feedback - can we just use the existing `applyAddOns` pattern and add the industry add-ons to the array instead of doing it separately?

Like, instead of adding line 44, change line 45 to be

```
  await applyAddOns(builder, ["generic-tasks", ...industryRoadmap.addOns, ...addOns]);
```

And same for modifications

Reply:
Unfortunately no, because its a collection of addon tasks already, and doesn't need to be imported the same way the addons need to be

Reply:
Oh gotcha, I understand. Makes sense